pom.xml

```diff
@@ -18,17 +18,35 @@
     <parent>
         <groupId>org.eclipse.ee4j</groupId>
         <artifactId>project</artifactId>
-        <version>1.0.6</version>
+        <version>1.0.7</version>
     </parent>
+
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.eclipse.angus</groupId>
     <artifactId>angus-activation-project</artifactId>
     <packaging>pom</packaging>
     <version>1.0.0-SNAPSHOT</version>
-    <name>Angus Activation project</name>
+    <name>Angus Activation Project</name>
     <description>${project.name}</description>
     <url>https://github.com/eclipse-ee4j/angus-activation</url>
 
+    <properties>
+        <spec.version>2.1</spec.version>
+        <spec.title>Jakarta Activation Specification</spec.title>
+        <activation.version>2.1.0-RC1</activation.version>
+        <activation.recompile.skip>false</activation.recompile.skip>
```

Comment:
this property is no longer needed, remove it as well as its usages

Reply:
Removed here and in maven-compiler-plugin config.